.github/workflows/deploy.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - develop
-  pull_request:
```

Comment:
I removed it because I thought we'd better publish the github pages only when `develop` branch is updated.